lms/envs/aws.py

```diff
@@ -578,6 +578,14 @@
             'schedule': datetime.timedelta(hours=ENV_TOKENS.get('THIRD_PARTY_AUTH_SAML_FETCH_PERIOD_HOURS', 24)),
         }
 
+    # FAKE EMAIL DOMAIN setting is used to generate an email for an automatically provisioned account in case
+    # it is not provided by IdP (which should'nt normally be the case for providers with automatic provisioning)
+    FAKE_EMAIL_DOMAIN = ENV_TOKENS.get('FAKE_EMAIL_DOMAIN', 'fake-email-domain.foo')
```

Comment:
@bradenmacdonald initial use case was to provide fake email - so I would keep it (if possible) to reduce code drift. So, I believe it's best to wait for edX opinion on it.